src/Tooling/Bit.Tooling.Templates/TodoTemplate/TodoTemplate/Web/Services/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -6,12 +6,12 @@ public static IServiceCollection AppServices(this IServiceCollection services)
         {
             services.AddScoped(sp =>
             {
-                HttpClient httpClient = new(sp.GetRequiredService<AppHttpClientHandler>()) { BaseAddress = new Uri("https://localhost:5001/api/") };
+                HttpClient httpClient = new(sp.GetRequiredService<TodoTemplateHttpClientHandler>()) { BaseAddress = new Uri("https://localhost:5001/api/") };
```

Comment:
We've IConfiguration & appsettings in both app & web projects. Please use ApiServerAddress instead of hard coded server URL @sajadkardel